android/modules/ui/src/java/ti/modules/titanium/ui/android/SnackbarProxy.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2017 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+package ti.modules.titanium.ui.android;
+
+import android.app.Activity;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import org.appcelerator.kroll.annotations.Kroll;
+import org.appcelerator.titanium.proxy.TiViewProxy;
+import org.appcelerator.titanium.view.TiUIView;
+
+import ti.modules.titanium.ui.widget.TiUISnackbar;
+
+@Kroll.proxy(creatableInModule = AndroidModule.class)
+public class SnackbarProxy extends TiViewProxy
+{
+	private static final String TAG = "SnackbarProxy";
+
+	@Kroll.constant
+	public static final int LENGTH_LONG = Snackbar.LENGTH_LONG;
+	@Kroll.constant
+	public static final int LENGTH_SHORT = Snackbar.LENGTH_SHORT;
+	@Kroll.constant
+	public static final int LENGTH_INDEFINITE = Snackbar.LENGTH_INDEFINITE;
+
+	private TiUISnackbar snackbar;
+
+	public SnackbarProxy()
+	{
+		super();
+	}
```

Comment:
Is this required?

Reply:
no, looks like I can remove this one though :smile: